README.md

```diff
@@ -21,3 +21,6 @@ corresponding [subfolder](https://github.com/5G-MAG/rt-mbms-examples/tree/develo
 
 ### Simple Express Server
 This simple HTTP server based on express.js can be used to statically host files. 
+
+More information can be found in the
+corresponding [subfolder](https://github.com/jordijoangimenez/rt-mbms-examples/tree/development/simple-express-server).
```

Comment:
This links to your fork "jordijoangimenez". We should link to our main repo

Reply:
Thanks, Daniel. fixed